src/configurations/destinations/azure_datalake/ui-config.json

```diff
@@ -150,8 +150,8 @@
           "type": "checkbox",
           "label": "Skip User Table",
           "value": "skipUsersTable",
-          "footerNote": "Enable this feature to send event data exclusively to the Identify table, and skip user table. Eliminates the need for merge operation on the user table",
-          "default": false
+          "footerNote": "Disable to create a Users table. The table store all unique users, but, because of merge operations, can increase warehouse operations time significantly.",
```

Comment:
Minor: `Disable the creation of a Users table. The table stores all unique users, but note that due to merge operations, it can significantly increase warehouse operation time.`

Applicable for all.